XCUITests/TranslationSnackBarTest.swift

```diff
@@ -5,15 +5,46 @@
 import XCTest
 
 class TranslationSnackBarTest: BaseTestCase {
-    func testSnackBar() {
+    
+    // // This test checks to the correct functionalty of the Translation prompt and Translation is done corrrectly using Google
+    func testSnackBarDisplayed() {
         userState.localeIsExpectedDifferent = true
         navigator.openURL(path(forTestPage: "manifesto-zh-CN.html"))
         waitUntilPageLoad()
         navigator.goto(TranslatePageMenu)
         waitForExistence(app.buttons["TranslationPrompt.doTranslate"])
+        app.buttons["TranslationPrompt.doTranslate"].tap()
+        waitForValueContains(app.textFields["url"], value: "translate.google")
+        if iPad(){
+                app.buttons["URLBarView.backButton"].tap()
+        } else{
+                app.buttons["TabToolbar.backButton"].tap()
+        }
+        app.buttons["TranslationPrompt.dontTranslate"].tap()
+        XCTAssertFalse(app.buttons["TranslationPrompt.dontTranslate"].exists)
     }
-
-    func testSetting() {
+    
+    // This test checks to see if Translation is enabled by default from the Settings menu and can be correctly disabled
+    func testSnackBarNotDispalyed() {
         navigator.goto(TranslationSettings)
+        let translationToggle = app.tables.cells.element(boundBy:0)
+        XCTAssertTrue(translationToggle.isEnabled)
+        translationToggle.tap()
```

Comment:
I would add an action `enableTraslationFeatureFromSetting` or whatever name you think is good for this in the FxScreenGraph `TranslationSettings`  [screen](https://github.com/mozilla-mobile/firefox-ios/blob/master/XCUITests/FxScreenGraph.swift#L550)